Library/Homebrew/utils/svn.rb

```diff
@@ -1,4 +1,8 @@
 module Utils
+  def self.clear_svn_version_cache
+    remove_instance_variable(:@svn) if instance_variable_defined?(:@svn)
```

Comment:
@reitermarkus I'm not sure how this gets resolved without adding a method here. What would you propose?